model/postlude/validate_config.sail

```diff
@@ -245,6 +245,14 @@ function check_extension_param_constraints() -> bool = {
   valid
 }
 
+function check_pointer_masking() -> bool = {
+  if (xlen == 32 & (hartSupports(Ext_Ssnpm) | hartSupports(Ext_Smmpm) | hartSupports(Ext_Smnpm))) then {
+    print_endline("Ssnmp, Smmpm or Smnpm (Pointer Masking) is enabled: these are not supported on RV32.");
```

Comment:
```suggestion
    print_endline("One or more Ssnmp, Smmpm or Smnpm (Pointer Masking) are enabled: these are not supported on RV32.");
```

Reply:
I'd also move this to `check_misc_extension_dependencies()` since this is a single `if`.